testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/suites/ManagementReadXmlAsFileTestCase.java

```diff
@@ -25,47 +23,33 @@
 import java.nio.file.Paths;
 import java.util.Collections;
 
-
 import org.apache.commons.io.IOUtils;
 import org.jboss.as.controller.client.Operation;
 import org.jboss.as.controller.client.OperationMessageHandler;
 import org.jboss.as.controller.client.OperationResponse;
 import org.jboss.as.controller.client.helpers.domain.DomainClient;
-import org.jboss.as.version.Stability;
+import org.jboss.as.test.integration.domain.management.util.WildFlyManagedConfiguration;
+import org.jboss.as.test.integration.management.util.CLIWrapper;
 import org.jboss.dmr.ModelNode;
-import org.junit.AfterClass;
 import org.junit.Assert;
 import org.junit.BeforeClass;
 import org.junit.Test;
-import org.wildfly.test.stability.StabilityDomainSetupSnapshotRestoreTasks;
 
 /**
  * Test of various read-config-as-xml-file against the domain controller.
  *
  * @author Emmanuel Hugonnet (c) 2022 Red Hat, Inc.
  */
-public class ManagementReadXmlCommunityTestCase extends ManagementReadXmlTestCase{
-    static StabilityDomainSetupSnapshotRestoreTasks tasks;
+public class ManagementReadXmlAsFileTestCase extends ManagementReadXmlTestCase {
```

Comment:
Just from our general discussion today, as this is not referenced by the Suite it may not be being executed in CI so worth a double check.

Reply:
I noticed that ManagementReadXmlTestCase wasn't referenced either.